app/enterprise/2.1.x/kong-for-kubernetes/deployment-options.md

```diff
@@ -3,7 +3,10 @@ title: Kong for Kubernetes deployment options
 skip_read_time: true
 ---
 <div class="alert alert-ee blue">
-<b>Coming Soon:</b> Kong for Kubernetes Enterprise (K4K8s) is not currently available for use with {{site.ee_product_name}} 2.1.x beta. It will be released within the beta timeframe.
+For the {{site.ee_product_name}} 2.1.x beta, Kong for Kubernetes Enterprise
```

Comment:
Is this just a temporary change for beta, or are you fully stopping development of the kong-enterprise-k8s image going forward?

Reply:
Pending customer feedback from the beta, but as of now, assuming no unexpected issues using the `kong-enterprise-edition` image, yes, we will end updates for `kong-enterprise-k8s` and direct existing users to switch images for the upgrade to 2.1 GA.